docs/oauth.md

```diff
@@ -104,19 +104,19 @@ Note: The keys in accessTokenResponseMaps and refreshTokenResponseMaps are Table
 | REFRESHTOKEN | Yes | No | Used by Tableau to get a new ACCESSTOKEN when the old one expired. |
 | username | Yes | No | Used by Tableau to identify the token. |
 | access-token-issue-time | No | No | Used to note when the token is issued, will default to the time when the token is sent back to Tableau if not set. |
-| access-token-expires-in | No | No | Use to note the validation time of your ACCESSTOKEN, will default to 3600s if not set. |
-| [your own field] | No | No | It is usually not needed to define your own field, and this field will not be able to participate in any oauth flow. |
+| access-token-expires-in | No | No | Use to note the validation time of your ACCESSTOKEN, will default to 3600 s if not set. |
```

Comment:
Not sure if `3600s` was indented to be a value example or not here. @zsun-tableau

Reply:
No it's not, in connection attr we store it like this without unit:  `access-token-expires-in = 3600`

I think it's better to say `will default to 3600 seconds if not set.` or `will default to 1 hour if not set.`